vision/noxfile.py

```diff
@@ -45,10 +45,6 @@ def blacken(session):
     """Run black.
 
     Format code to uniform standard.
-    
-    This currently uses Python 3.6 due to the automated Kokoro run of synthtool.
```

Comment:
Is synthtool up to date on your machine? It seems maybe the template is the prev. version?

Reply:
Ah yep it needed updating, done.